forte/processors/nltk_processors.py

```diff
@@ -111,3 +112,24 @@ def _process(self, input_pack: DataPack):
                 end_pos = begin_pos + len(sentence_text)
                 sentence_entry = Sentence(input_pack, begin_pos, end_pos)
                 input_pack.add_or_get_entry(sentence_entry)
+
+
+class NLTKNER(PackProcessor):
+    r"""A wrapper of NLTK NER.
+    """
+    def __init__(self):
+        super().__init__()
+        self.token_component = None
+
+    def _process(self, input_pack: DataPack):
+        for sentence in input_pack.get(Sentence):
+            token_entries = list(input_pack.get(entry_type=Token,
+                                                range_annotation=sentence,
+                                                component=self.token_component))
+            tokens = [(token.text, token.pos) for token in token_entries]
+            ne_tree = ne_chunk(tokens)
+
+            ners = [chunk.label() if hasattr(chunk, 'label') else ''
+                    for chunk in ne_tree]
+            for token, ner in zip(token_entries, ners):
+                token.set_fields(ner=ner)
```

Comment:
Similar to the chunking case, setting another attribute to the token does not fit into Forte's span based design principle. You can find a proper implementation here.